### ISSUE TITLE
 Class cache.store does not exist

### ISSUE THREAD

**Issue by JackNoordhuis** [laravel/framework#22079]
- Laravel Version: 5.5.14
- PHP Version: 7.0.25
- Database Driver & Version: N/A

### Description:
Error with my project when running any artisan commands.

### Steps To Reproduce:
Run any artisan command.

Error: `[ReflectionException]             
  Class cache.store does not exist`

Trace:
```
[2017-11-15 22:47:09] production.ERROR: Class cache.store does not exist {"exception":"[object] (ReflectionException(code: -1): Class cache.store does not exist at /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php:752)
[stacktrace]
#0 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(752): ReflectionClass->__construct('cache.store')
#1 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(631): Illuminate\\Container\\Container->build('cache.store')
#2 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(586): Illuminate\\Container\\Container->resolve('cache.store', Array)
#3 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(721): Illuminate\\Container\\Container->make('cache.store', Array)
#4 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(885): Illuminate\\Foundation\\Application->make('cache.store')
#5 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(813): Illuminate\\Container\\Container->resolveClass(Object(ReflectionParameter))
#6 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(780): Illuminate\\Container\\Container->resolveDependencies(Array)
#7 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(631): Illuminate\\Container\\Container->build('Illuminate\\\\Cons...')
#8 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(586): Illuminate\\Container\\Container->resolve('Illuminate\\\\Cons...', Array)
#9 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(721): Illuminate\\Container\\Container->make('Illuminate\\\\Cons...', Array)
#10 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Console/Scheduling/Schedule.php(36): Illuminate\\Foundation\\Application->make('Illuminate\\\\Cons...')
#11 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(101): Illuminate\\Console\\Scheduling\\Schedule->__construct()
#12 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(749): Illuminate\\Foundation\\Console\\Kernel->Illuminate\\Foundation\\Console\\{closure}(Object(Illuminate\\Foundation\\Application), Array)
#13 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(631): Illuminate\\Container\\Container->build(Object(Closure))
#14 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Container/Container.php(586): Illuminate\\Container\\Container->resolve('Illuminate\\\\Cons...', Array)
#15 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(721): Illuminate\\Container\\Container->make('Illuminate\\\\Cons...', Array)
#16 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(104): Illuminate\\Foundation\\Application->make('Illuminate\\\\Cons...')
#17 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(89): Illuminate\\Foundation\\Console\\Kernel->defineConsoleSchedule()
#18 [internal function]: Illuminate\\Foundation\\Console\\Kernel->Illuminate\\Foundation\\Console\\{closure}(Object(Illuminate\\Foundation\\Application))
#19 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(820): call_user_func(Object(Closure), Object(Illuminate\\Foundation\\Application))
#20 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(769): Illuminate\\Foundation\\Application->fireAppCallbacks(Array)
#21 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/BootProviders.php(17): Illuminate\\Foundation\\Application->boot()
#22 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(213): Illuminate\\Foundation\\Bootstrap\\BootProviders->bootstrap(Object(Illuminate\\Foundation\\Application))
#23 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(296): Illuminate\\Foundation\\Application->bootstrapWith(Array)
#24 /Users/Jack/Stuff/PHPClassroom/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(119): Illuminate\\Foundation\\Console\\Kernel->bootstrap()
#25 /Users/Jack/Stuff/PHPClassroom/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#26 {main}
"} 
```

**Comment by themsaid** [comment]
Can't replicate on a fresh laravel installation, could be something in your app.

**Comment by JackNoordhuis** [comment]
I couldn't replicate it on a fresh install either. I've spent the better part of 4 hours trying to debug the issue to no avail, and seeing as though I couldn't get a stacktrace with any useful information as to where the problem is caused I resulted to opening an issue.

Any pointers as to what would cause the issue would be much appreciated as not even a google search seems to yield a solution for this.
